statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/ResourceHandler.java

```diff
@@ -85,7 +85,7 @@ protected void verifyStateMachineArnIsPresent(String resourceArn) throws AmazonS
         }
     }
 
-    private AmazonServiceException getStateMachineDoesNotExistException() {
+    protected AmazonServiceException getStateMachineDoesNotExistException() {
```

Comment:
does this part need to be reverted?

Reply:
This is expected, we need access to the method in both ResourceHandler and DeleteHandler.